harvard-huit/Harvard-Geospatial-Library
# Dockerfile update that makes the container run as App user in non-local environments

**Dockerfile update that makes the container run as App user in non-local environments**
* * *

**JIRA Ticket**: https://jira.huit.harvard.edu/browse/HGL-399


# What does this Pull Request do?
This has already been deployed to the Dev server. The container is currently running as app, and it is logging to the files correctly.

# How should this be tested?

A description of what steps someone could take to:
* Running this command on the dev server that the container is running on, you should see it running as "app": ```docker inspect $(docker ps -q) --format '{{.Config.User}} {{.Name}}'```
* The log files are also located at /logs/gbl/geoblacklight/development.log. You should see it updating as you browse the application

# Interested parties
@awoods @ives1227
